core/src/main/java/org/infinispan/remoting/transport/jgroups/EmbeddedJGroupsChannelConfigurator.java

```diff
@@ -196,6 +206,11 @@ private static int findProtocol(String protocol, List<ProtocolConfiguration> sta
    public enum StackCombine {
       COMBINE,
       INSERT_AFTER,
+      INSERT_ABOVE,
+      INSERT_BEFORE,
+      INSERT_BELOW,
+      ABOVE,
+      BELOW,
```

Comment:
`ABOVE` and `BELOW` aren't used or included in the `.xsd`

Reply:
removed them,